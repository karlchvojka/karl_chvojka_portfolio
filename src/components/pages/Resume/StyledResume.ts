// Library Imports
import styled from 'styled-components'
import media from 'globalConfig/mobile'

// Asset Imports
import bg from 'globalAssets/images/hexrepeat2.jpg'
import { defaultColors } from 'globalConfig/colors'

const StyledResume = styled.section`
  /** FONT RESET */
  background-color: "#ffffff";
  background-image: none;
  margin: 0 auto;
  max-width: 80%;
  

  h1 {
    color: ${defaultColors.cyberBlue};
    font-family: 'MontserratBold', sans-serif;
    font-size:21px;
    font-weight: bold;
    margin-bottom: 20px;
  }

  h2 {
    color: ${defaultColors.cyberBlue};
    font-family: 'MontserratBold', sans-serif;
    font-size:18px;
    margin-bottom: 20px;
    margin-top: 20px;
    text-transform: uppercase;
    line-height: 1;
  }

  h3 {
    color: ${defaultColors.cyberBlue};
    font-family: 'MontserratBold', sans-serif;
    font-size:16px;
    font-weight: bold;
    margin-bottom: 20px;
    text-transform: uppercase;
    line-height: 1;
  }

  h4 {
    color: ${defaultColors.cyberBlue};
    font-family: 'Montserrat', sans-serif;
    font-size:15px;
    font-weight: 700;
    margin-bottom:20px;
    text-transform: uppercase;
    line-height: 1;
  }

  h5 {
    color: ${defaultColors.cyberBlue};
    font-family: 'Montserrat', sans-serif;
    font-size:15px;
    text-transform: uppercase;
    line-height: 1;
  }

  h6 {
    color: ${defaultColors.cyberBlue};
    font-family: 'Montserrat', sans-serif;
    font-size:15px;
    text-transform: uppercase;
    line-height: 1;
  }

  p {
    color: ${defaultColors.cyberBlue};
    font-family: 'Montserrat', sans-serif;
    font-size:14px;
    line-height:18px;
    margin-bottom:10px;
  }

  a:link, a:visited {
    color: ${defaultColors.orange};
    font-family: 'Montserrat', sans-serif;
    font-size:14px;
    text-decoration: none;
  }

  a:hover {
    color: ${defaultColors.cyberBlue};
  }

  /* Min width of 576 */
  ${media.landscapePhones `
    h1 {
      font-size: 24px;
    }

    h2 {
      font-size: 20px;
    }

    h3 {
      font-size: 20px;
    }

    h4 {
      font-size: 18px;
    }

    h5 {
      font-size: 18px;
    }

    h6 {
      font-size: 18px;
    }

    p {
      font-size: 14px;
    }

    a:link, a:visited {
      font-size: 14px;
    }
  `}

  /* Min width of 768 */
  ${media.tablet `
    h1 {
      font-size: 26px;
    }

    h2 {
      font-size: 22px;
    }

    h3 {
      font-size: 22px;
    }

    h4 {
      font-size: 20px;
    }

    h5 {
      font-size: 20px;
    }

    h6 {
      font-size: 20px;
    }

    p {
      font-size: 16px;
      line-height: 20px;
    }

    a:link, a:visited {
      font-size: 16px;
    }
  `}

  /* Min width of 1024 */
  ${media.desktop `
    h1 {
      font-size: 35px;
    }

    h2 {
      font-size: 32px;
    }

    h3 {
      font-size: 30px;
    }

    h4 {
      font-size: 28px;
    }

    h5 {
      font-size: 26px;
    }

    h6 {
      font-size: 24px;
    }

    p {
      font-size: 16px;
      line-height: 20px;
    }

    a:link, a:visited {
      font-size: 16px;
    }
  `}

  /* Min width of 1200 */
  ${media.widescreen `
    p {
      font-size: 18px;
      line-height:22px;
    }

    a:link, a:visited {
      font-size: 18px;
    }
  `}

  /* Min width of 1980 */
  ${media.udh `
    p {}
    a:link, a:visited {}
  `}

  body {
    background-color: #ffffff !important;
  }

  @page {
    size: auto;   /* auto is the initial value */
    margin-top: 5mm;  /* this affects the margin in the printer settings */
    margin-bottom: 5mm; 
  }

  min-width: 100%;

  h1, h2 {
    font-family: 'Roboto Condensed', sans-serif;
    font-weight: 700;
  }

  p, a:link {
    font-family: 'Lato';
  }

  h1 {
    margin: 0px;
    text-transform: uppercase;
  }

  h2 {
    color: rgb(0, 171, 223);
    font-family: "Roboto Condensed", sans-serif;
    margin-bottom: 20px;
  }

  p {
    color: #000000;
  }

  .contacts, .profile, .workExp, .skills, .education {
    padding: 20px;
  }

  .controls {
    background-color: #000000;
    display: grid;
    grid-template-columns: 75% 100px;
    justify-content: space-between;
    padding: 5.5em 1em 1em 1em;
    
    p {
      color: #ffffff;
      font-family: 'Montserrat';
      margin-bottom: 0px;
    }

    a {
      font-weight: bold;
    }
  }

  .contacts {
    border-bottom: 5px solid ${defaultColors.cyberBlue};
    position: relative;

    h2 {
      color: #000000;
      margin: 5px 0px;
      padding-bottom: 5px;
      border-bottom: 1px solid ${defaultColors.cyberBlue};
      border-top: 1px solid ${defaultColors.cyberBlue};
      width: fit-content;
      padding-top: 7px;
    }

    p, a:link { 
      text-transform: uppercase;
    }

    p {
      color: #000000;
      font-weight: 700;
      line-height: 1;
    }

    a:link {
      font-family: 'Roboto Condensed', sans-serif;
      font-weight: 400;
    }
  }

  .contacts * {
    position: relative;
  }

  .contacts::before {    
    content: "";
    background-image: url(${bg});
    position: absolute;
    top: 0px;
    right: 0px;
    bottom: 0px;
    left: 0px;
    opacity: 0.5;
  }

  .profile {
    border-bottom: 1px solid ${defaultColors.cyberBlue};
    padding-bottom: 15px;
  }

  .workExp {
    border-bottom: 1px solid ${defaultColors.cyberBlue};

    .workplace {
      margin-bottom: 20px;

      h3 {
        font-family: "Roboto Condensed", sans-serif;
        margin-bottom: 0.5rem;
      }

      .workplaceHeader {
        display: grid;
        grid-template-columns: 1fr 1fr;
        justify-content: space-between;

        h4 {
          color: #000000;
          font-family: "Roboto Condensed", sans-serif;
          margin-bottom: 0px;
        }
        p {
          font-family: "Roboto Condensed", sans-serif;
          font-weight: 700;
          margin-bottom: 0px;
          text-transform: uppercase;
          text-align: right;
        }
      }

      p.keywords {
        font-size: 0.8rem;
        font-style: italic;
        line-height: 1rem;
        margin-bottom: 1rem;

        span {
          font-weight: 700;
        }
      }
    }
  }

  .skills {
    border-bottom: 1px solid ${defaultColors.cyberBlue};

    .skillsWrap {
      column-gap: 20px;

      section {
        margin-top: 20px;
      
        h3 {
          font-family: "Roboto Condensed", sans-serif;
        }
      }
    }
  }

  .education {
    border-bottom: 1px solid ${defaultColors.cyberBlue};
    
    .edItem {
      h3 {
        font-family: "Roboto Condensed", sans-serif;
        margin-bottom: 0.5rem;
      }

      .edHeader {
        display: grid;
        grid-template-columns: 1fr 1fr;
        justify-content: space-between;

        h4 {
          color: #000000;
          font-family: "Roboto Condensed", sans-serif;
          margin-bottom: 0px;
        }

        p {
          font-family: "Roboto Condensed", sans-serif;
          font-weight: 700;
          margin-bottom: 0px;
          text-transform: uppercase;
          text-align: right;
        }
      }
    }
  }

  /* Min width of 1024 */
    ${media.desktop `
    `}

  @media print {
    #Header, #Footer, header, .header { 
      display: none !important; 
    }
    .contacts, .profile, .workExp, .skills, .education {
      padding: 0px 0px 20px 0px;
    }

    .controls {
      display:none;
    }
  }

  #Header, #Footer, header, .header { 
    display: none !important; 
  }
`

export default StyledResume
