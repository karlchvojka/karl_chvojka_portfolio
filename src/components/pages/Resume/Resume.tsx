// Framework Imports
import React from 'react'

// Library Imports
import { Link } from "react-router-dom";

// CSS Imports
import StyledResume from './StyledResume'

// Data Imports
import resume from 'globalAssets/data/resume'
import skills from 'globalAssets/data/skills'

/**
 * This the Resume component.
 */
const Resume = () => {
  return (
    <StyledResume>
      <section className='controls'>
        <p>To save as pdf:&nbsp;&nbsp; 1. Right click, click 'Print'&nbsp;&nbsp; 2. Change Destination to 'Save as PDF'&nbsp;&nbsp; 3. Press Save</p>
        <Link to="/">Go Back</Link>
      </section>
      <section className='contacts'>
        <h1>Karl Chvojka</h1>
        <h2>Front End Web Developer</h2>
        <p>Portfolio: <a href="http://karlchvojka.com">karlchvojka.com</a></p>
        <p>Github: <a href="https://github.com/karlchvojka">https://github.com/karlchvojka</a></p>
        <p>Linkedin: <a href="https://www.linkedin.com/in/karlchvojka/">https://www.linkedin.com/in/karlchvojka/</a></p>
        <p>Email: <a href="mailto:karl.chvojka@protonmail.com">karl.chvojka@protonmail.com</a></p>
      </section>
      <section className='profile'>
        <h2>Profile</h2>
        <p>I am a Fullstack Web Developer with 10 Years of industry experience building websites, being part of advertising campaigns, interacting with traditional media, and working in teams of people from a variety of disciplines. I specialize in Frontend Technologies (JavaScript, Responsive design libraries, React, HTML5, CSS3) with knowledge and understanding of backend Technologies (Node.js,  Ruby on Rails, Relational and Document databases like PostgreSQL, MongoDB, mySQL, etc). </p>
        <p>I also have experience in Project Management, project budgeting, SEO techniques, Online and Traditional advertising, web, graphic, and traditional media design, and how all the parts can work together in an advertising campaign.</p>
      </section>
      <section className='workExp'>
        <h2>Work Experience</h2>
        {
          resume.workExperience.placements.map(( place, index) => (
            <section className='workplace'>
              <h3>{place.company}</h3>
              <section className='workplaceHeader'>
                <h4>{place.position}</h4>
                <p>{place.dates}</p>
              </section>
              <p className='keywords'><span>Noteable Keywords:</span> { place.keywords.map((keyword, index) => (`${keyword}, ` )) }</p>
              <p>{place.desc}</p>
            </section>
          ))
        }
      </section>
      <section className='skills'>
        <h2>Notable Skills</h2>
        <section className='skillsWrap'>
          {
            skills.map(( section, index ) => (
              <section className={section.title.replace(/\s/g, '')} key={section.key + index}>
                <h3>{section.title}</h3>
                <p>
                  {section.skillList.map((skill, index) => (<span>{skill.title + ', '}</span>))}
                </p>
              </section>
            ))
          }
        </section>
      </section>
      <section className='education'>
        <h2>Education</h2>
        <section className='edItem'>
          <h3>Lighthouse Labs, Toronto</h3>
          <section className='edHeader'>
            <h4>Web Development Bootcamp</h4>
            <p>APR 2019 - JUNE 2019</p>
          </section>
          <p>The Web Development Bootcamp is a 12 week, 40+ hours/week bootcamp. It covers the most popular and marketable languages and frameworks used in Web Development today.</p>
          <p>The topics covered are: JavaScript, Node.js, Express, PostgreSQL, MongoDB, React, Ruby on Rails, Automated Testing, and Computer Science Fundamentals.</p>
        </section>
        <section className='edItem'>
          <h3>EDMONTON ARTS COLLEGE</h3>
          <section className='edHeader'>
            <h4>DIGITAL MEDIA PRODUCTION PROGRAM</h4>
            <p>June 2010</p>
          </section>
          <p>Topics Covered: Education, Student Instruction, Front End Development (HTML5, CSS3), JavaScript, jQuery, Wordpress, Drupal, Joomla!, Design Interpretation, Career Guidence</p>
        </section>
      </section>
    </StyledResume>
  )
}

export default Resume
