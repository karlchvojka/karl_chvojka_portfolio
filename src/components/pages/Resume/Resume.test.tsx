// Framework Imports
import React from 'react'
import renderer from 'react-test-renderer'

// Library Imports
import { render, screen, fireEvent, act } from '@testing-library/react'
import { describe, expect } from '@jest/globals'
import '@testing-library/jest-dom/'

// Component Imports
import Resume from './Resume'

describe("(Component) Column Layout", () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Resume />).toJSON();
    expect(domTree).toMatchSnapshot();
  })
})