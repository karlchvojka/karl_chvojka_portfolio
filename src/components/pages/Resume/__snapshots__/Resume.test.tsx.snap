// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Column Layout Matches DOM Snapshot 1`] = `
<section
  className="sc-beqWaB iMNEMy"
>
  <section
    className="controls"
  >
    <p>
      To save as pdf:   1. Right click, click 'Print'   2. Change Destination to 'Save as PDF'   3. Press Save
    </p>
  </section>
  <section
    className="contacts"
  >
    <h1>
      Karl Chvojka
    </h1>
    <h2>
      Front End Web Developer
    </h2>
    <p>
      Portfolio: 
      <a
        href="http://karlchvojka.com"
      >
        karlchvojka.com
      </a>
    </p>
    <p>
      Github: 
      <a
        href="https://github.com/karlchvojka"
      >
        https://github.com/karlchvojka
      </a>
    </p>
    <p>
      Linkedin: 
      <a
        href="https://www.linkedin.com/in/karlchvojka/"
      >
        https://www.linkedin.com/in/karlchvojka/
      </a>
    </p>
    <p>
      Email: 
      <a
        href="mailto:karl.chvojka@protonmail.com"
      >
        karl.chvojka@protonmail.com
      </a>
    </p>
  </section>
  <section
    className="profile"
  >
    <h2>
      Profile
    </h2>
    <p>
      I am a Fullstack Web Developer with 10 Years of industry experience building websites, being part of advertising campaigns, interacting with traditional media, and working in teams of people from a variety of disciplines. I specialize in Frontend Technologies (JavaScript, Responsive design libraries, React, HTML5, CSS3) with knowledge and understanding of backend Technologies (Node.js,  Ruby on Rails, Relational and Document databases like PostgreSQL, MongoDB, mySQL, etc).
    </p>
    <p>
      I also have experience in Project Management, project budgeting, SEO techniques, Online and Traditional advertising, web, graphic, and traditional media design, and how all the parts can work together in an advertising campaign.
    </p>
  </section>
  <section
    className="workExp"
  >
    <h2>
      Work Experience
    </h2>
    <section
      className="workplace"
    >
      <h3>
        Open Mind Developments
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          Front-End Developer
        </h4>
        <p>
          Nov 2022 - Dec 2022
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Front-End Development (HTML5, CSS3, CSS Grid, Flexbox)
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Tailwind
          , 
        </span>
        <span>
          Shopify
          , 
        </span>
        <span>
          ESLint
          , 
        </span>
        <span>
          JSX
          , 
        </span>
        <span>
          Developement Operations
          , 
        </span>
        <span>
          Gitlab CI/CD
          , 
        </span>
      </p>
      <p>
        Front-End Development focusing on Customer interaction with the Shopify platform.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        Kepler Communications
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          Web Application Developer
        </h4>
        <p>
          Mar 2021 - Jul 2022
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Front-End Development (HTML5, CSS3, CSS Grid, Flexbox)
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          ReactJS
          , 
        </span>
        <span>
          Cypress.io
          , 
        </span>
        <span>
          Jest
          , 
        </span>
        <span>
          React Testing Library
          , 
        </span>
        <span>
          ESLint
          , 
        </span>
        <span>
          JSX
          , 
        </span>
        <span>
          Developement Operations
          , 
        </span>
        <span>
          Gitlab CI/CD
          , 
        </span>
        <span>
          Semantic Ui
          , 
        </span>
      </p>
      <p>
        Front-End developer building interfaces for handling Communication Satellite monitoring and configuration, and configuring continuous integration, deployments and testing for each.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        Lighthouse Labs
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          Front-End Developer
        </h4>
        <p>
          Aug 2019 - Mar 2021
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Customer Service
          , 
        </span>
        <span>
          Student Mentorship
          , 
        </span>
        <span>
          Project Management
          , 
        </span>
        <span>
          Marketing
          , 
        </span>
        <span>
          Front-End Development (HTML5, CSS3, CSS Grid, Flexbox)
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          ReactJS
          , 
        </span>
        <span>
          Ruby On Rails
          , 
        </span>
        <span>
          JSX
          , 
        </span>
        <span>
          Active Record
          , 
        </span>
      </p>
      <p>
        Working with the Marketing team to plan and execute the building of Static ReactJs sites for marketing and promotional purposes. Working with the internal staff to maintain and upgrade internal systems in React, Ruby on Rails, etc.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        FREELANCE WORK
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          FULL STACK DEVELOPER
        </h4>
        <p>
          AUG 2010 - PRESENT
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Business Management
          , 
        </span>
        <span>
          Client Interaction
          , 
        </span>
        <span>
          Client relations
          , 
        </span>
        <span>
          Customer service
          , 
        </span>
        <span>
          Project Management
          , 
        </span>
        <span>
          Self Employment
          , 
        </span>
        <span>
          Front-End Development (HTML5, CSS3)
          , 
        </span>
        <span>
          Wordpress
          , 
        </span>
        <span>
          PHP
          , 
        </span>
        <span>
          MySQL
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          Photography
          , 
        </span>
        <span>
          User Guide Creation
          , 
        </span>
      </p>
      <p>
        Managing and completing websites for a variety of clients. Projects are handled from initial contact with the client to the final launch including Concepts, Wireframing, Design, Development, Migration, Content Management, Testing, and Launching. Supplying clients with Hosting and Host maintenance, including Server updates, and Hosting provider interaction when needed.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        FREECODECAMP
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          GLOBAL COMMUNITY MANAGER
        </h4>
        <p>
          AUG 2017 - APR 2020
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Social Media Management
          , 
        </span>
        <span>
          Facebook Group moderation
          , 
        </span>
        <span>
          Community Organizing
          , 
        </span>
        <span>
          Public speaking
          , 
        </span>
      </p>
      <p>
        FreeCodeCamp is an online learning resource for Fullstack Development learning. FreeCodeCamp has broken learning Web Development into digestible chunks to allow people to learn how to code in their spare time. As a Global Community Manager, my biggest input into the FCC ecosystem is managing the moderators for the Facebook Groups and other social media.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        MEDIAFACE, TORONTO
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          FULLSTACK WEB DEVELOPER
        </h4>
        <p>
          JUNE 2017 - OCT 2017
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Client Research
          , 
        </span>
        <span>
          Client relations
          , 
        </span>
        <span>
          Front-End Development (HTML5, CSS3)
          , 
        </span>
        <span>
          Wordpress
          , 
        </span>
        <span>
          PHP
          , 
        </span>
        <span>
          MySQL
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          Technical SEO Implementation
          , 
        </span>
      </p>
      <p>
        Building Responsive Worpdress Themes with Bootstrap. Tasks included building full websites, and Wordpress Plugins. As well as setting up SEO Tracking codes for Google Analytics, Facebook, and Bing.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        DIGITAL SHIFT MARKETING, TORONTO
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          FRONT-END DEVELOPER
        </h4>
        <p>
          FEB 2016 - OCT 2016
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Development Team Management Training
          , 
        </span>
        <span>
          Process writing
          , 
        </span>
        <span>
          Client Research
          , 
        </span>
        <span>
          Client relations
          , 
        </span>
        <span>
          Customer service
          , 
        </span>
        <span>
          Project Management
          , 
        </span>
        <span>
          Front End Development (HTML5, CSS3)
          , 
        </span>
        <span>
          Wordpress
          , 
        </span>
        <span>
          PHP
          , 
        </span>
        <span>
          MySQL
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          Technical SEO Implementation
          , 
        </span>
      </p>
      <p>
        Building Responsive Worpdress Themes with Bootstrap. Tasks included building full websites to building landing pages for SEO A/B Testing. As well as setting up SEO Tracking codes for Google Analytics, Facebook, and Bing. Management duties included heading projects, as well as writing processes and procedures to be used by the development department and managing the IT needs of the company.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        VENTUREWEB, TORONTO
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          WEB CONSULTANT, FRONT END DEVELOPER
        </h4>
        <p>
          SEPT 2013 – DEC 2014
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Development Team Management Training
          , 
        </span>
        <span>
          Process writing
          , 
        </span>
        <span>
          Client Research
          , 
        </span>
        <span>
          Client relations
          , 
        </span>
        <span>
          Customer service Project Management
          , 
        </span>
        <span>
          Front-End Development (HTML5, CSS3)
          , 
        </span>
        <span>
          Wordpress
          , 
        </span>
        <span>
          PHP
          , 
        </span>
        <span>
          MySQL
          , 
        </span>
        <span>
          Javascript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          Bootstrap
          , 
        </span>
        <span>
          Technical SEO Implementation
          , 
        </span>
      </p>
      <p>
        Working as a web consultant through VentureWeb for Accenture, as a site manager for Accenture's Careers Websites. My tasks included building full websites to building landing pages for SEO A/B Testing. As well as setting up SEO Tracking codes for Google Analytics, Facebook, and Bing.
      </p>
    </section>
    <section
      className="workplace"
    >
      <h3>
        NORTHLANDS, EDMONTON
      </h3>
      <section
        className="workplaceHeader"
      >
        <h4>
          SR. FRONT END DEVELOPER
        </h4>
        <p>
          SEPT 2012 – APR 2013
        </p>
      </section>
      <p
        className="keywords"
      >
        <span>
          Noteable Keywords:
        </span>
         
        <span>
          Front-End Development (HTML5, CSS3)
          , 
        </span>
        <span>
          JavaScript
          , 
        </span>
        <span>
          jQuery
          , 
        </span>
        <span>
          PHP
          , 
        </span>
        <span>
          Wordpress
          , 
        </span>
        <span>
          Drupal
          , 
        </span>
        <span>
          Joomla!
          , 
        </span>
        <span>
          User guide creation
          , 
        </span>
        <span>
          Content Management
          , 
        </span>
        <span>
          Web Platform Maintenance
          , 
        </span>
        <span>
          Server Maintenance
          , 
        </span>
        <span>
          Enterprise Level Company
          , 
        </span>
      </p>
      <p>
        Primary responsibility for position was the updating and maintenance of the 13 web properties for Northlands. Acted as the primary contact for the hosting companies, and IT service providers that Northlands works with. This included managing, updating and maintaining the web servers, as well as working with the hosting companies to coordinate hardware maintenance and downtime. Additionally, responsible for internal system creation for managing new content to be added to the web properties.
      </p>
    </section>
  </section>
  <section
    className="skills"
  >
    <h2>
      Notable Skills
    </h2>
    <section
      className="skillsWrap"
    >
      <section
        className="Languages"
      >
        <h3>
          Languages
        </h3>
        <p>
          <span>
            HTML5, 
          </span>
          <span>
            CSS3, 
          </span>
          <span>
            JavaScript, 
          </span>
          <span>
            TypeScript, 
          </span>
          <span>
            ES6, 
          </span>
          <span>
            Liquid, 
          </span>
        </p>
      </section>
      <section
        className="Libraries"
      >
        <h3>
          Libraries
        </h3>
        <p>
          <span>
            NodeJS, 
          </span>
          <span>
            Express.Js, 
          </span>
          <span>
            ReactJS, 
          </span>
          <span>
            Webpack 5, 
          </span>
          <span>
            Babel, 
          </span>
          <span>
            ESLint, 
          </span>
          <span>
            Styled-Components, 
          </span>
          <span>
            React Testing Library, 
          </span>
          <span>
            Jest, 
          </span>
          <span>
            Cypress.io, 
          </span>
          <span>
            Semantic Ui, 
          </span>
          <span>
            Ajax, 
          </span>
          <span>
            Bootstrap, 
          </span>
          <span>
            JQuery, 
          </span>
          <span>
            Tailwind CSS, 
          </span>
          <span>
            Shopify, 
          </span>
        </p>
      </section>
      <section
        className="DatabaseSystems"
      >
        <h3>
          Database Systems
        </h3>
        <p>
          <span>
            MySQL, 
          </span>
          <span>
            Postgressql, 
          </span>
          <span>
            MongoDb, 
          </span>
          <span>
            Restful Apis, 
          </span>
        </p>
      </section>
    </section>
  </section>
  <section
    className="education"
  >
    <h2>
      Education
    </h2>
    <section
      className="edItem"
    >
      <h3>
        Lighthouse Labs, Toronto
      </h3>
      <section
        className="edHeader"
      >
        <h4>
          Web Development Bootcamp
        </h4>
        <p>
          APR 2019 - JUNE 2019
        </p>
      </section>
      <p>
        The Web Development Bootcamp is a 12 week, 40+ hours/week bootcamp. It covers the most popular and marketable languages and frameworks used in Web Development today.
      </p>
      <p>
        The topics covered are: JavaScript, Node.js, Express, PostgreSQL, MongoDB, React, Ruby on Rails, Automated Testing, and Computer Science Fundamentals.
      </p>
    </section>
    <section
      className="edItem"
    >
      <h3>
        EDMONTON ARTS COLLEGE
      </h3>
      <section
        className="edHeader"
      >
        <h4>
          DIGITAL MEDIA PRODUCTION PROGRAM
        </h4>
        <p>
          June 2010
        </p>
      </section>
      <p>
        Digital Media production based in Graphic Design.
      </p>
      <p>
        Topics Covered: Education, Student Instruction, Front End Development (HTML5, CSS3), JavaScript, jQuery, Wordpress, Drupal, Joomla!, Design Interpretation, Career Guidence
      </p>
    </section>
  </section>
</section>
`;
